tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+The ``ACRN-Crashlog`` is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
+
+Building
+********
+
+Build dependencies
+==================
+
+The ``ACRN-Crashlog`` tool depends on the following libraries (build and runtime):
+
+- libevent
+
+- OpenSSL
+
+- libxml2
+
+- systemd
+
+- telemetrics-client-dev (optional, detected at build time)
```

Comment:
Below is says telemetrics-client is required, but that's on on your list?

Reply:
Hmm. There is a history: we supposed it only run on Clear Linux before, then some compiling issues comes out on Ubuntu. We added some patch for the compiling issues, which will detect if telemetrics exists. So for  building, telemetrics-client-dev is optional here. Please refer patch 05927b1592b27fb3667a0a9d92abc6e4b1e16d85. Geoffroy helped to add the guide in GSG.